flink-table/flink-table-planner/src/main/java/org/apache/calcite/avatica/util/DateTimeUtils.java

```diff
@@ -26,7 +26,8 @@
 import java.util.TimeZone;
 
 /*
- * THIS FILE HAS BEEN COPIED FROM THE APACHE CALCITE PROJECT UNTIL CALCITE-1884 IS FIXED.
+ * THIS FILE HAS BEEN COPIED FROM THE APACHE CALCITE PROJECT UNTIL CALCITE-2989 IS FIXED.
```

Comment:
@danny0405, per the Flink table contribution guideline. when we alter Calcite classes, we need to have a `[CALCITE-xxxx]` ticket explained the future consolidation plan. as well as explicitly comment out the changes made on top of the copied Calcite classes. 

This class is copied from avatica 1.14.0 release and made only the following lines of changes.
please refer to CALCITE-2989 / FLINK-11935 for more info